packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -0,0 +1,167 @@
+import moment from "moment"
+import { getDownvoteRatio } from "../../components/sunshineDashboard/UsersReviewInfoCard"
+import { AutoRateLimit, RateLimitInfo, RecentKarmaInfo, TimeframeUnitType, UserKarmaInfo, UserRateLimit, rateLimitThresholds } from "./types"
+import { userIsAdmin, userIsMemberOf } from "../vulcan-users"
+import { RecentVoteInfo } from "../../server/repos/VotesRepo"
```

Comment:
This needs to be `import type` for  this import to not blow up if anything on the client imports anything from this file